src/app/shared/services/contacts.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+
+export interface ContactsInfo {
+  class: string;
+  title: string;
+  content: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ContactsService {
+  private contactsInfo: ContactsInfo [] = [
```

Comment:
@Yuriy-27 just for the future I would suggest you to use `json-server` npm module.
Just try it and you can imitate API which contains your data like contacts etc. And you will have practice to work with HTTP requests in this project